freeipa.tf

```diff
@@ -3,6 +3,31 @@
 #-------------------------------------------------------------------------------
 
 locals {
+  # Get Shared Services account ID from the default provider
+  this_account_id = data.aws_caller_identity.sharedservices.account_id
+
+  # Look up Shared Services account name from AWS organizations
+  # provider
+  this_account_name = [
+    for account in data.aws_organizations_organization.cool.accounts :
+    account.name
+    if account.id == local.this_account_id
+  ][0]
+
+  # Determine Shared Services account type based on account name.
+  #
+  # The account name format is "ACCOUNT_NAME (ACCOUNT_TYPE)" - for
+  # example, "Shared Services (Production)".
+  this_account_type = length(regexall("\\(([^()]*)\\)", local.this_account_name)) == 1 ? regex("\\(([^()]*)\\)", local.this_account_name)[0] : "Unknown"
```

Comment:
Yo dawg.  I heard you like parenthesizes.  So I put parenthesizes in your parenthesizes. 
I didn't know we were using Lisp in the COOL.  Nifty.
t